src/main/webapp/app/exam/participate/exam-start-information/exam-start-information.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { ArtemisSharedComponentModule } from 'app/shared/components/shared-component.module';
+import { InformationBox, InformationBoxComponent } from 'app/shared/information-box/information-box.component';
+import { Exam } from 'app/entities/exam.model';
+import { StudentExam } from 'app/entities/student-exam.model';
+import { ArtemisExamSharedModule } from 'app/exam/shared/exam-shared.module';
+import dayjs from 'dayjs/esm';
+
+@Component({
+    selector: 'jhi-exam-start-information',
+    standalone: true,
+    imports: [ArtemisSharedModule, ArtemisSharedComponentModule, InformationBoxComponent, ArtemisExamSharedModule],
+    templateUrl: './exam-start-information.component.html',
+    styleUrl: './exam-start-information.component.scss',
+})
+export class ExamStartInformationComponent implements OnInit {
+    examInformationBoxData: InformationBox[] = [];
+
+    @Input() exam: Exam;
+    @Input() studentExam: StudentExam;
+
+    totalPoints: number | undefined;
+    totalWorkingTimeInMinutes: number | undefined;
+    moduleNumber: string | undefined;
+    courseName: string | undefined;
+    examiner: string | undefined;
+    numberOfExercisesInExam: number | undefined;
+    examinedStudent: string | undefined;
+    startDate: dayjs.Dayjs | undefined;
+    gracePeriodInMinutes: number | undefined;
+
```

Comment:
I think | undefined is fine, but consider using the `?` to make it more compact: `totalPoints?: number`